vividus-studio-server/vividus-studio-plugin/src/main/java/org/vividus/studio/plugin/VividusStudioActivator.java

```diff
@@ -0,0 +1,42 @@
+/*-
+ * =========================LICENSE_START==================================
+ * Vividus Studio :: Plugin
+ * %%
+ * Copyright (C) 2020 Vividus Framework
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
```

Comment:
it should be equal to this https://github.com/vividus-framework/vividus-build-system/blob/master/1.0/config/spotless/spotless.license.java (only year may be different)